examples/DotNetCore/WebDemoWithEventHub/WebDemoWithEventHub/EventHubService.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Azure.Identity;
+using Azure.Messaging.EventHubs;
+using Azure.Messaging.EventHubs.Consumer;
+using Azure.Messaging.EventHubs.Processor;
+using Azure.Storage.Blobs;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+using Microsoft.Extensions.Options;
+using System;
+using System.Threading.Tasks;
+
+namespace WebDemoWithEventHub
+{
+    public class EventHubService
+    {
+        private Settings _settings;
+
+        private EventHubConnection _eventHubConnection;
+
+        private EventProcessorClient _processorClient;
+
+        private IConfigRefresher _configRefresher;
+
+        public EventHubService(IConfiguration config, IConfigRefresher configRefresher)
+        {
+            _configRefresher = configRefresher;
+
+            _eventHubConnection = config.GetSection("WebDemo:Connection").Get<EventHubConnection>();
```

Comment:
I would name it "WebDemo:EventHubConnection" to match what you have in code.

Reply:
Renamed.

---
In reply to: [427596123](https://github.com/Azure/AppConfiguration/pull/329#discussion_r427596123) [](ancestors = 427596123)

Reply:
I don't see the code change.

---
In reply to: [428450341](https://github.com/Azure/AppConfiguration/pull/329#discussion_r428450341) [](ancestors = 428450341,427596123)

Reply:
Redid the change.

---
In reply to: [432186230](https://github.com/Azure/AppConfiguration/pull/329#discussion_r432186230) [](ancestors = 432186230,428450341,427596123)